docs/tempo/website/configuration/_index.md

```diff
@@ -173,6 +174,89 @@ ingester:
     [ complete_block_timeout: <duration>]
 ```
 
+## Metrics-generator
+For more information on configuration options, see [here](https://github.com/grafana/tempo/blob/main/modules/generator/config.go).
+
+The metrics-generator processes spans and write metrics to a Prometheus-compatible database.
+
+The metrics-generator is an optional component, it can be enabled by setting the following top-level setting.
+In microservices mode, it must be set for the distributors and the metrics-generators.
```

Comment:
what does the metrics generator do if metrics_generator_enabled = false? :)

Reply:
We log a warning if `--target="metrics-generator"` and `metrics_generator_enabled` is not set. The metrics-generator will run just as normal, but it won't receive any data from the distributors.

https://github.com/grafana/tempo/blob/0ad50642247809bf841b31b6721890d5e90d060d/cmd/tempo/app/app.go#L137-L140

To be fair, it's valid to only set `metrics_generator_enabled` in the distributor. This flag was originally part of the `distributor` config block and we moved it to the top-level to make it more visible.
I'm not sure if it's worth making this distinction, I mostly copied this section because we also have it for the search flags.